### DISCUSSION TITLE
About github student developer pack application

### DISCUSSION THREAD

**Discussion by Novrule** [community/community#66896] [GitHub Education]
### Hi everyone,

![화면 캡처 2023-09-13 005345](https://github.com/NaranggeSaida/TIL/assets/119842443/beeaacad-df4b-49ab-850a-98940229614b)

I previously applied for the GitHub Student Developer Pack and received an email stating that my application was rejected. So, I tried to apply again, but I encountered the message shown in the image above, preventing me from applying. What should I do?

**Comment by khuramgill** [comment]
If you previously applied for the GitHub Student Developer Pack and your application was rejected, you might encounter difficulties reapplying. To address this issue and reapply successfully, you should take the following steps:

1. **Review the Rejection Email**: Start by reviewing the email you received when your application was rejected. This email may contain information about why your application was denied, and it could offer guidance on what you need to do to be eligible for approval. Addressing the reasons for your initial rejection is crucial.

2. **Wait for the Required Time Period**: GitHub might have a waiting period or cooldown period before you can reapply after a rejection. Ensure that you have waited the necessary amount of time before attempting to reapply. The email you received or the GitHub Student Developer Pack documentation might mention this waiting period.

3. **Contact GitHub Support**: If you believe your rejection was in error or if you've addressed the reasons for your initial rejection, you can try contacting GitHub Support for assistance. They can provide guidance on reapplying and help resolve any issues with your application. You can typically reach GitHub Support through their support platform or contact form on the GitHub website.

4. **Ensure Eligibility**: Make sure you meet all the eligibility criteria for the GitHub Student Developer Pack. This usually includes being a student at an accredited educational institution and having a verified education-related email address. Double-check that your GitHub account and email address meet these requirements.

5. **Clear Browser Cache and Cookies**: Sometimes, issues with reapplying can be related to browser caching or cookies. Try clearing your browser's cache and cookies or using a different browser to submit your application.

6. **Use a Different Email Address**: If your previous rejection was associated with a particular email address, you might consider using a different, eligible email address to reapply.

7. **Review Documentation**: Read through the GitHub Student Developer Pack documentation and guidelines carefully to ensure you are following all the necessary steps for applying. Make sure you are using the correct link or portal to apply.

8. **Patience and Persistence**: If you are sure you meet the eligibility criteria and have addressed the reasons for your initial rejection, but you still encounter issues, be patient and persistent. Contacting GitHub Support for assistance is often the best course of action in such cases.

Remember that the GitHub Student Developer Pack is a program with specific eligibility criteria, and it's essential to follow their guidelines and requirements when applying or reapplying.
